_posts/2023-10-11-specify-file-audit-log-encoding.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: 'Change the default encoding of the audit log file in WildFly 29'
+date: 2023-10-11
+tags: audit-logging encoding
+synopsis:  An overview of how to change the default encoding of the audit log file.
+author: ivassile
+---
+
+The default encoding used for the audit log file is UTF-8. You can change the default encoding by specifying `encoding` attribute in `file-audit-log`, `periodic-rotating-file-audit-log` or `size-rotating-file-audit-log` elements in the Elytron subsystem. 
+Possible values are: `UTF-8`, `UTF-16BE`, `UTF-16LE`, `UTF-16`, `US-ASCII` or `ISO-8859-1`.
+
+[source,xml]
+----
+<file-audit-log name="local-file" path="audit.log" relative-to="jboss.home.dir" synchronized="false" autoflush="true" format="JSON" encoding="UTF-16"/>
```

Comment:
Would you be able to show CLI commands here instead of the XML configuration?

Reply:
Added CLI command to the "Example" section.